eyelid-match-ops/benches/match-ops.rs

```diff
@@ -229,7 +234,13 @@ pub fn bench_inv(settings: &mut Criterion) {
 
     let rng = rand::thread_rng();
 
-    let p = sample::<FULL_RES_POLY_DEGREE>(rng);
+    let params = YasheParams {
+        t: 1024,
+        delta: 3.2,
+    };
+    let ctx: Yashe<FULL_RES_POLY_DEGREE> = Yashe::new(params);
```

Comment:
Let's benchmark key generation as well.

This will need a change to the benchmark code, and an extra `|key` in the benchmark selection pattern here:
https://github.com/Inversed-Tech/eyelid/blob/facb2447c66dffe9b297cc729a8706b944fa9d84/.github/workflows/ci-bench-changes.yml#L55

Reply:
This is now ticket #72.